### PR TITLE
Update Travis-CI

### PR DESCRIPTION
**Pull request by ocefpaf**
I also simplified the packaging a little bit.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,54 @@
...
+    env: TEST_TARGET=latest_compliance_checker
+  - python: 3.6
+    env: TEST_TARGET=latest_compliance_checker
+  allow_failures:
+  - python: 3.6
+    env: TEST_TARGET=coding_standards
+  - python: 2.7
+    env: TEST_TARGET=latest_compliance_checker
+  - python: 3.6
+    env: TEST_TARGET=latest_compliance_checker
```
Comment (ocefpaf) [inline .travis.yml]: Tests against `compliance-checker` `master` to catch errors early.

File: .travis.yml [hunk 2]
```diff
@@ -1,19 +1,54 @@
...
 
 before_install:
-    - wget http://bit.ly/miniconda -O miniconda.sh
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export PATH="$HOME/miniconda/bin:$PATH"
-    - conda config --add channels ioos -f
```
Comment (ocefpaf) [inline .travis.yml]: :scream: 

Changed to `conda-forge`.

File: .travis.yml [hunk 3]
```diff
@@ -1,19 +1,54 @@
...
+    fi
+
+  - if [[ $TEST_TARGET == 'latest_compliance_checker' ]]; then
+      conda uninstall compliance-checker ;
+      pip install git+https://github.com/ioos/compliance-checker.git ;
+      py.test -s -rxs -v ;
+    fi
+
+  - if [[ $TEST_TARGET == 'coding_standards' ]]; then
+      find . -type f -name "*.py" | xargs flake8 --max-line-length=100 --ignore W391 ;
```
Comment (ocefpaf) [inline .travis.yml]: Added a `coding_standard` test.

File: MANIFEST.in [hunk 4]
```diff
@@ -1,5 +1,5 @@
+include *.txt
 include VERSION
 include LICENSE
 include README.md
 recursive-include cc_plugin_ugrid *.py
-recursive-include requirements *.txt
```
Comment (ocefpaf) [inline MANIFEST.in]: No need for to sets of requirement files. `conda` and `pip` should be compatible, if not we should fix the conda package.

**Comment by brianmckenna** [comment]
@ocefpaf with the repo name updated to cc-plugin-ugrid want to make sure I didn't break anything

**Comment by ocefpaf** [comment]
All green! The only failure is coding standards but that is in the `allow_failures` section: https://travis-ci.org/ioos/cc-checker-ugrid/jobs/287776067

**Pull request merged**